romeo/asset-contract/contracts/asset.clar

```diff
@@ -15,6 +15,7 @@
 ;; constants
 ;;
 (define-constant err-forbidden (err u403))
+(define-constant err-amount-zero (err u404))
```

Comment:
err 404 is usually used for not found errors.

Reply:
I see this mimicking the http errors, I think can use 400 that is **Bad Request**